jupyter_publishing_service/authenticator/jwt_authenticator.py

```diff
@@ -34,7 +36,7 @@ class JWTAuthenticator(LoggingConfigurable):
     ).tag(config=True)
 
     async def fetch_public_keys(self):
-        context = create_ssl_context(verify=self.ssl_cert_file)
+        context = False  # create_ssl_context(verify=self.ssl_cert_file)
```

Comment:
is this an intentional change or temporary?

Reply:
No, this is probably temporary. Although, we shouldn't need to create the ssl cert outselves. This should be setup separately by the operator.